src/ziggurat/streams.clj

```diff
@@ -92,6 +97,22 @@
         additional-tags  {:topic_name topic-entity-name}]
     (.transform stream-builder (transformer-supplier metric-namespace oldest-processed-message-in-s additional-tags) (into-array [(.name (store-supplier-builder))]))))
 
+(defn- traced-handler-fn [handler-fn channels message-payload topic-entity]
+  (let [parent-ctx (TracingKafkaUtils/extractSpanContext (:headers message-payload) tracer)
```

Comment:
right now the headers contain only the tracing headers. Will this still work when other information is added to the kafka headers?

Reply:
yes this will work with all kafka headers